test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/apiKeys.test.js

```diff
@@ -38,6 +38,14 @@ describe('#compileApiKeys()', () => {
           value: 'valueForKey9876543211',
           customerId: 'customerid98765',
         },
+        {
```

Comment:
As mentioned in the original issue, all new tests should be using `runServerless`. The old ones are quite fragile and we're actively migrating off of them - hence the decision to not accept additional technical debt in form of extension of old tests. Related documentation: https://github.com/serverless/serverless/blob/master/test/README.md#unit-tests

Reply:
@pgrzesik understood! I read the unit test documentation for runServerless and can add the new tests for it. Actually, can you clarify two things for me. I did notice that it mentioned we can update old tests with new runServerless implementation, so should I update these older tests for apiKeys with the new runServerless way or add my own tests with runServerless? Also, want to make sure that this file is okay to add the test into.

Reply:
Yes, please add the tests to this file under a new `describe` section :+1: 

As for updating older tests - if you'd like to do that it would be awesome, but it's also understood if you'd prefer to only test out the introduced change with new tests and leave the old ones untouched for now.